ERP/service/telerikerpService/telerikerpService.csproj

```diff
@@ -135,7 +135,7 @@
       <HintPath>..\packages\Microsoft.Data.Edm.5.8.1\lib\net40\Microsoft.Data.Edm.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.Data.OData">
-      <HintPath>..\packages\Microsoft.Data.OData.5.8.1\lib\net40\Microsoft.Data.OData.dll</HintPath>
+      <HintPath>..\packages\Microsoft.Data.OData.5.8.3\lib\net40\Microsoft.Data.OData.dll</HintPath>
```

Comment:
Is there a reason why this one couldn't go all the way to .4?

Reply:
No other reason than a typo. It's fixed now.